script/popup/index.js

```diff
@@ -156,16 +156,6 @@ $(document).ready(function(){
         $("#username").text("@" + localStorage['githubusername'] );
     }
 
-    if(!isweb3()){
```

Comment:
itd be great if this still worked in chrome.. i just think we should remove it from firefox so we can cut a release there.